2010-06-10  Andreas Schwab  <schwab@redhat.com>

	* sysdeps/unix/sysv/linux/getpagesize.c: Don't assume AT_PAGESIZE
	is always available.

--- a/sysdeps/unix/sysv/linux/getpagesize.c
+++ b/sysdeps/unix/sysv/linux/getpagesize.c
@@ -28,7 +28,7 @@
 int
 __getpagesize ()
 {
-#ifdef __ASSUME_AT_PAGESIZE
+#if 0 && defined __ASSUME_AT_PAGESIZE
   assert (GLRO(dl_pagesize) != 0);
   return GLRO(dl_pagesize);
 #else
-- 
1.7.2

